### ISSUE TITLE
Visualize quality with sonarQube

### ISSUE THREAD

**Issue by bengreenier** [bengreenier/browserd#11]
We want to be able to visualize our code quality overtime, using [sonarqube](https://www.sonarqube.org/).

We'll create a non-yaml pipeline to run this as PR validation, alongside our existing yaml pipeline. Why no yaml? Because sonarqube is a nice to have tool for us as part of the core team, but not something we think anyone else would necessarily benefit from setting up - not to mention it isn't a quick setup process. 

__TL;DR: sonarQube will serve as a sort of "internal" tool for us, despite the results being public.__
